.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/jasonephraim/apprentice-react-o-shop/src/App.js":"1"},{"size":528,"mtime":1609775330304,"results":"2","hashOfConfig":"3"},{"filePath":"4","messages":"5","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"12mgxbv","/Users/jasonephraim/apprentice-react-o-shop/src/App.js",[]]
```

Comment:
could you add this file to .gitignore. For reference, here is a good guide to determine what files should be added to .gitignore: https://zellwk.com/blog/gitignore/

Reply:
Thanks - .eslintcache added to .gitignore